buildSrc/src/main/java/Dep.kt

```diff
@@ -1,20 +1,24 @@
 package io.github.droidkaigi.feeder
 
+object Versions {
+    const val compose = "1.0.0-beta01"
+}
+
 object Dep {
     object Compose {
         const val activity = "androidx.activity:activity-compose:1.3.0-alpha02"
         const val constraintLayout =
             "androidx.constraintlayout:constraintlayout-compose:1.0.0-alpha03"
         const val navigation = "androidx.navigation:navigation-compose:1.0.0-alpha08"
-        const val runtime = "androidx.compose.runtime:runtime:1.0.0-beta01"
-        const val livedata = "androidx.compose.runtime:runtime-livedata:1.0.0-beta01"
-        const val foundation = "androidx.compose.foundation:foundation:1.0.0-beta01"
-        const val layout = "androidx.compose.foundation:foundation-layout:1.0.0-beta01"
-        const val ui = "androidx.compose.ui:ui:1.0.0-beta01"
-        const val tooling = "androidx.compose.ui:ui-tooling:1.0.0-beta01"
-        const val material = "androidx.compose.material:material:1.0.0-beta01"
-        const val iconsExtended = "androidx.compose.material:material-icons-extended:1.0.0-beta01"
-        const val animation = "androidx.compose.animation:animation:1.0.0-beta01"
+        const val runtime = "androidx.compose.runtime:runtime:${Versions.compose}"
+        const val livedata = "androidx.compose.runtime:runtime-livedata:${Versions.compose}"
+        const val foundation = "androidx.compose.foundation:foundation:${Versions.compose}"
+        const val layout = "androidx.compose.foundation:foundation-layout:${Versions.compose}"
+        const val ui = "androidx.compose.ui:ui:${Versions.compose}"
+        const val tooling = "androidx.compose.ui:ui-tooling:${Versions.compose}"
+        const val material = "androidx.compose.material:material:${Versions.compose}1"
+        const val iconsExtended = "androidx.compose.material:material-icons-extended:${Versions.compose}"
```

Comment:
I believe putting a new line after `=` fix the warning by reviewdog

Reply:
@risako070310 Could you apply ktlint for a line ?
```
Step 'ktlint' found problem in 'buildSrc/src/main/java/Dep.kt':

> Task :spotlessKotlin FAILED
Error on line: 20, column: 1
Exceeded max line length (100)
java.lang.AssertionError: Error on line: 20, column: 1
Exceeded max line length (100)
```